SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -393,6 +395,24 @@ void ActorSystem::Advance()
 
 	CommandsHandler.ProcessOps(*ActorSubView->GetViewDelta().WorkerMessages);
 
+	for (const Worker_Op& Op : *ActorSubView->GetViewDelta().WorkerMessages)
+	{
+		if (Op.op_type == WORKER_OP_TYPE_COMMAND_RESPONSE)
+		{
+			const Worker_CommandResponseOp& CommandResponse = Op.op.command_response;
+			Worker_EntityId_Key EntityId = SpatialConstants::INVALID_ENTITY_ID;
```

Comment:
const?

Reply:
Need to set it in the `RemoveAndCopy` below.